plugin/src/main/resources/issues/property.jelly

```diff
@@ -23,25 +23,29 @@
           <colgroup>
             <col class="col-width-5"/>
             <col class="col-width-1"/>
+            <col class="col-width-1"/>
             <col class="col-width-6"/>
```

Comment:
The total should sum up to `12`. 

```suggestion
            <col class="col-width-1"/>
            <col class="col-width-6"/>
```

Reply:
Fixed: d0cb13dfb330064aba876d0fc690b42d5cbfd57c 
Where does this value come from though?